### PR TITLE
fix reading stream came from db

### PR DESCRIPTION
**Pull request by StalkAlex**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #13501

### CODE REVIEW

**Review by samdark — APPROVED** [review]
Looks good. Need a line for changelog.

File: framework/rbac/DbManager.php [hunk 1]
```diff
@@ -631,7 +631,12 @@ public function getRule($name)
             ->from($this->ruleTable)
             ->where(['name' => $name])
             ->one($this->db);
-        return $row === false ? null : unserialize($row['data']);
+        $result = $row['data'];
```
Comment (cebe) [inline framework/rbac/DbManager.php]: `$row` can be false, so that check must be done here.

**Review by cebe — CHANGES_REQUESTED** [review]
Also a testcase would be great to cover this.

File: framework/rbac/DbManager.php (hunk 1 above)
Comment (StalkAlex) [inline framework/rbac/DbManager.php]: ok, I'll change it and make test.

**Comment by StalkAlex** [comment]
I fixed both getRule and getRules method. Some tests were failed due to this bug, now there are all ok. Not sure what else testcase could be here, because current tests have covered this already from my point of view.

**Comment by samdark** [comment]
2.0.11 was not failing: https://travis-ci.org/yiisoft/yii2/builds/197358860

**Comment by StalkAlex** [comment]
I ran locally only rbac group for 2.0.11.1. Here's result:
default
I suppose it's correct behaviour because of bug I reported.
Here's run for fixed version:
default

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**